src/nih_wayfinding/app/scripts/profiles/profile-preference-options.js

```diff
@@ -39,13 +39,13 @@
 
             speedOpts: [{
                 text: 'About average (2.5 mph)',
-                value: 2.5
+                value: 1.1176
```

Comment:
Also will have to delete and create new users for the related calculations to display properly in the UI